types/components/Input.d.ts

```diff
@@ -0,0 +1,40 @@
+import { GeneratedPropTypes } from '../types';
+import React from 'react';
+export declare type InputProps = {
+    as?: any;
+    button?: any;
+    disabled?: boolean;
+    hint?: string;
+    icon?: string;
+    onBlur?: Function;
+    onFocus?: Function;
+    prefix?: string;
+    rows?: number | string;
+    suffix?: string;
+    withError?: boolean;
+    wrapperProps?: GeneratedPropTypes;
+} & {
+    [key: string]: any;
+};
+export declare const Input: React.ForwardRefExoticComponent<
+    Omit<
+        {
+            as?: any;
+            button?: any;
+            disabled?: boolean | undefined;
+            hint?: string | undefined;
+            icon?: string | undefined;
+            onBlur?: Function | undefined;
+            onFocus?: Function | undefined;
+            prefix?: string | undefined;
+            rows?: string | number | undefined;
+            suffix?: string | undefined;
+            withError?: boolean | undefined;
+            wrapperProps?: GeneratedPropTypes | undefined;
```

Comment:
it's undefined already
```suggestion
            disabled?: boolean;
            hint?: string;
            icon?: string;
            onBlur?: Function;
            onFocus?: Function;
            prefix?: string;
            rows?: string | number;
            suffix?: string;
            withError?: boolean;
            wrapperProps?: GeneratedPropTypes;
```